airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/asserts.py

```diff
@@ -62,3 +67,7 @@ def verify_records_schema(
             stream_errors[record.stream][str(error.schema_path)] = error
 
     return stream_errors
+
+
+def is_int(checker, value):
+    return isinstance(value, int)
```

Comment:
```suggestion
```
Is this still used anywhere? Looks leftover

Reply:
Thanks for spotting this :)